blockchain/blockchain.go

```diff
@@ -454,6 +454,36 @@ func (bc *BlockChain) SetProposerPolicy(val uint64) {
 	bc.chainConfig.Istanbul.ProposerPolicy = val
 }
 
+func (bc *BlockChain) SetLowerBoundBaseFee(val uint64) {
+	bc.chainConfigMu.RLock()
+	defer bc.chainConfigMu.RUnlock()
+	bc.chainConfig.Governance.KIP71.LowerBoundBaseFee = val
```

Comment:
`chainConfig.Governance.KIP71` can be `nil`, if it wasn't set at the first place.
For instance, when `genesis.json` file don't contain the "kip71" at all.

Please check if the new Set...() methods work using homi or relevant unit tests.